src/SFA.DAS.LevyTransferMatching.Web/Controllers/PledgesController.cs

```diff
@@ -101,5 +101,12 @@ public async Task<IActionResult> Level(LevelPostRequest request)
             await _orchestrator.UpdateCacheItem(request);
             return RedirectToAction("Create", new CreateRequest() { EncodedAccountId = request.EncodedAccountId, CacheKey = request.CacheKey });
         }
-   }
+
+        [HttpGet]
+        [Route("{pledgeId}/confirmation")]
+        public async Task<IActionResult> Confirmation(string pledgeId, string encodedAccountId)
```

Comment:
I'd rather we removed the two parameters here in favour of a single object representing the request, eg. "ConfirmationRequest".